docs/standard/frameworks.md

```diff
@@ -22,21 +22,21 @@ For package-based target frameworks (for example, .NET 5+, .NET Core, and .NET S
 
 The following table defines the most common target frameworks, how they're referenced, and which version of [.NET Standard](net-standard.md) they implement. These target framework versions are the latest stable versions. Prerelease versions aren't shown. A target framework moniker (TFM) is a standardized token format for specifying the target framework of a .NET app or library.
 
-| Target framework      | Latest <br/> stable version | Target framework moniker (TFM) | Implemented <br/> .NET Standard version |
-| :-: | :-: | :-: | :-: |
-| .NET 6                | 6                           | net6.0                         | N/A                                     |
-| .NET 5                | 5                           | net5.0                         | N/A                                     |
-| .NET Standard         | 2.1                         | netstandard2.1                 | N/A                                     |
-| .NET Core             | 3.1                         | netcoreapp3.1                  | 2.1                                     |
-| .NET Framework        | 4.8                         | net48                          | 2.0                                     |
+| Target framework | Latest <br/> stable version | Target framework moniker (TFM) | Implemented <br/> .NET Standard version |
+|:----------------:|:---------------------------:|:------------------------------:|:---------------------------------------:|
+| .NET 6           | 6                           | net6.0                         | N/A                                     |
```

Comment:
I'm not seeing a .NET 7 row added to this table, is that intentional?

Reply:
Yes, this table doesn't show prerelease versions.